### PR TITLE
Update guide

### PR DESCRIPTION
**Pull request by skkumaravel**
- Add Conclusion
- Change Intro
- Add code pull

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -143,3 +155,9 @@ First we will ensure that we are able to stream data, then direct the data to ou
 8. Change the `Metric` to `in-octets` and `Aggregator` to `last` and check the `Rate` checkbox
 
 9. Click the refresh icon on the top right of the box, you should now see data!
+
+
+
+## Conclusion
+
+Now that you have Streaming Telemetry working on IOS-XE, try making it work with NX-OS and IOS-XR using the learning labs and other guides as reference. This environment is meant for you to be able to play around with the different Cisco operating systems and create your own collectors for databases you use in your environment.
```
Comment (remingtonc) [inline getting_started.md]: IOS XR

File: getting_started.md [hunk 2]
```diff
@@ -28,9 +30,19 @@ Running on the Ubuntu server are:
    # Password: Cisco1234!
    ssh telemetry@10.10.20.25
    ```
-2. Start the Docker Stacks  
+
+2. Download the latest code for the project
+
    ```bash
    cd telemetry_stacks
+   git pull
```
Comment (remingtonc) [inline getting_started.md:38]: Just double checking - it's pre-provisioned with this cloned?

File: getting_started.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ This environment consists of several virtualized devices and software components
 * [Cisco Nexus 9000v](https://developer.cisco.com/docs/nx-os/#!developer-tooling/developer-tooling) (NX-OS)
 * Ubuntu Server
 
+***Note:** Streaming Telemetry on IOS XRv and Cisco Nexus 9000v are outside the scope of Streaming Telemetry 101, but there for you to to develop around in the Sandbox.
```
Comment (remingtonc) [inline getting_started.md:13]: Funny how that extra `*` makes everything bolded in GitHub's diffs.

**Review by remingtonc — APPROVED** [review]
Looks good, but IOS-XR -> IOS XR is only change requested.

File: getting_started.md (hunk 2 above)
Comment (skkumaravel) [reply getting_started.md:38]: It is pre-provisioned but if we ever make changes in the future, this way we don't need to go into the box

**Pull request merged**